client/src/partials/finance/reports/agedCreditors/agedCreditors.html

```diff
@@ -0,0 +1,33 @@
+<div class="flex-header static">
+  <div class="bhima-title">
+    <ol class="headercrumb">
+      <li class="static">{{ "TREE.FINANCE" | translate }}</li>
```

Comment:
All new client-side modules should use the `translate` directive instead of the `translate` filter.  Please update this code (and all other `translate` filters) to be `<element translate>KEY</element>`.